day04/ex02/ex02.md

```diff
@@ -0,0 +1,48 @@
+# Exercise 02 - Information Gain
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turn-in directory :   |  ex02              |
+|   Files to turn in :    |  information_gain.py           |
+|   Forbidden modules :   |  sklearn            |
+|   Forbidden functions : |  Anything that computes directly the functions. |
+|   Remarks :             |  Read the doc      |
+
+
+**Objective:**
+You must implement the following formula as a function: 
+$$
+IG(q) = S_0 - \sum_{i = 1}^{q} \frac{n_i}{N} S_i 
+$$
+
+Where
+- $X$ is a vector of dimension N * 1
```

Comment:
I guess there is a coherence with the previous definition of terms in the formula in ex00 and ex01. But it seems to me that the vocabulary change here, it might be confusing